test/compose/ca/cacert.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -x
+
+KEY_SIZE=2048
+
+rm -rf ca
+rm /certs/*
+mkdir -p ca/ca.db.certs
+touch ca/ca.db.index
+echo "1234" > ca/ca.db.serial
+openssl genrsa -des3 -passout pass:foobar -out ca/ca.key $KEY_SIZE || exit 1
+
+openssl req -new -x509 -passin pass:foobar \
+        -subj "/C=US/ST=CA/O=ACME/CN=acme.fake/emailAddress=wcoyote@acme.fake" \
```

Comment:
I got that reference
![image](https://github.com/AdRoll/erlmld/assets/17753587/2a571393-1a89-4493-82bb-66960474bd17)